components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js

```
@@ -171,8 +249,10 @@ class D2LEvaluationHubActivitiesList extends mixinBehaviors([D2L.PolymerBehavior
 		try {
 			var result = await this._parseActivities(entity);
 			this._data = result;
+			return this._loadSorts(entity);
```

Made these changes so we actually log errors instead of swallowing them

---

I think we should remove `loadSorts` here and add it to the `observer`: https://github.com/BrightspaceHypermediaComponents/activities/pull/59/files#diff-c36f46a8beb24cb46d7a213237cccab8L134